design/ceph/object/zone.md

```diff
@@ -80,9 +81,17 @@ After an ceph-object-store is configured to be in this ceph-object-zone, the all
 
 At the moment creating an ceph-object-zone resource only handles Day 1 initial configuration for the zone.
 
-Changes made to the resource's configuration or deletion of the resource are not reflected on the Ceph cluster.
+By default when a CephObjectZone is deleted or modified, the pools supporting the zone are not deleted from the Ceph cluster.
 
-To be clear, when the ceph-object-zone resource is deleted or modified, the zone is not deleted from the Ceph cluster. Zone deletion must be done through the toolbox.
+If user disables `preservePoolsOnDelete` then zone or pool will be removed only if all the objectstores belong to this zone need to deleted first.
+
+### Deleting ObjectStore belong to a multisite
+
+If the `objectstore` belong to [master zone](/design/ceph/object/ceph-multisite-overview.md/#master-zone) then deletion will be blocked until following is satisfied:
+- either all the peer zone need to removed first or choose another zone as master as steps mentioned [here](https://docs.ceph.com/en/latest/radosgw/multisite/#failover-and-disaster-recovery)
+- if no more peers attached then user need to remove the residing data.
+
+If the object store belong to peer zone, then it is directly removed, assuming data is synced and safely hold in other peers.
```

Comment:
```suggestion
If the object store belongs to a peer zone, then the deletion is not blocked and the zone is immediately deleted. If the data is still required, it will still be available in the peer cluster.
```